packages/api/models/rules.js

```diff
@@ -242,8 +242,8 @@ class Rule extends Manager {
       newRuleItem.state = 'ENABLED';
     }
 
-    newRuleItem.createdAt = Date.now();
-    newRuleItem.updatedAt = Date.now();
+    newRuleItem.createdAt = createdAt || Date.now();
+    newRuleItem.updatedAt = updatedAt || Date.now();
```

Comment:
These changes should be unit tested in the rules model tests

Reply:
[`8aedf3f` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/8aedf3f318e5c41e867a5bf29ac0642f85c06c0f)